Jenkinsfile

```diff
@@ -97,7 +97,7 @@ if (utils.isCI()){
     def witApiURL = 'https://api.openshift.io/'
```

Comment:
shouldn't the `witApiURL` env var be passed into `pipeline.deploy(..)`?

Reply:
Yes, obviously, sorry, fixed